samples/cpp/common/utils/include/samples/args_helper.hpp

```diff
@@ -45,5 +45,45 @@ void configurePrePostProcessing(std::shared_ptr<ov::Model>& function,
                                 const std::string& ioml);
 
 void printInputAndOutputsInfo(const ov::Model& network);
-void printInputAndOutputsInfoShort(const ov::Model& network);
-ov::element::Type getPrecision2(const std::string& value);
\ No newline at end of file
+ov::element::Type getPrecision2(const std::string& value);
+
+template <class T>
+void printInputAndOutputsInfoShort(const T& network) {
+    std::cout << "Network inputs:" << std::endl;
+    for (auto&& input : network.inputs()) {
+        std::string in_name = "***NO_NAME***";
+        std::string node_name = "***NO_NAME***";
+
+        // Workaround for "tensor has no name" issue
+        try {
+            in_name = input.get_any_name();
+        } catch (const ov::Exception&) {
+        }
+        try {
+            node_name = input.get_node()->get_friendly_name();
+        } catch (const ov::Exception&) {
+        }
+
+        std::cout << "    " << in_name << " (node: " << node_name << ") : " << input.get_element_type() << " / "
+                  << ov::layout::get_layout(input).to_string() << std::endl;
+    }
+
+    std::cout << "Network outputs:" << std::endl;
+    for (auto&& output : network.outputs()) {
+        std::string out_name = "***NO_NAME***";
+        std::string node_name = "***NO_NAME***";
+
+        // Workaround for "tensor has no name" issue
+        try {
+            out_name = output.get_any_name();
+        } catch (const ov::Exception&) {
+        }
+        try {
+            node_name = output.get_node()->get_input_node_ptr(0)->get_friendly_name();
```

Comment:
@ilyachur is it possible that this statement throws an exception?

Reply:
not sure... better be on the safe side. I think that sometimes get_friendly() name might throw (not sure about this particular node name)